### PR TITLE
Blog - Linux Text Processing Command

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

File: blog/_posts/2023-06-10-linux-text-processing-commands.md [hunk 1]
```diff
@@ -0,0 +1,1019 @@
...
+
+Output:
+
+~~~{ caption="Output"}
+HELLO WORLD
+~~~
+
+In the above command we have taken the output of the first command and then passed it through the `tr` command which capitalized all the letters.
+
+Let's take another example of installing [docker](/blog/rails-with-docker) on our system with the easy endpoint that Docker provide:
```
Comment (jalletto) [inline blog/_posts/2023-06-10-linux-text-processing-commands.md:69]:
```suggestion
Let's take another example of installing docker on our system with the easy endpoint that Docker provide:
```

File: blog/_posts/2023-06-10-linux-text-processing-commands.md [hunk 2]
```diff
@@ -0,0 +1,1019 @@
...
+
+The above command retrieves all the links from a markdown document. The link in a standard Markdown looks as shown below:
+
+~~~{ caption="Output"}
+[Earthly](https://earthly.dev)
+~~~
+
+So,the above command retrieves the link by looking for the pattern that matches how a link is embedded in a standard markdown.
+You can further use this to check if links are accessible or not.
+
```
Comment (jalletto) [inline blog/_posts/2023-06-10-linux-text-processing-commands.md:621]:
```suggestion
<div class="notice--info">
We have a whole post on [Understanding Awk](/blog/awk-examples/) if you'd like to learn more.
</div>
```

Comment (jalletto) [reply blog/_posts/2023-06-10-linux-text-processing-commands.md:621]: Can you add this and confirm it looks ok locally?

File: blog/_posts/2023-06-10-linux-text-processing-commands.md [hunk 3]
```diff
@@ -0,0 +1,1019 @@
...
+~~~{.bash caption=">_"}
+$ sed 's/k8s/kubernetes/' k8s.md 
+~~~
+
+Output:
+
+~~~{ caption="Output"}
+kubernetes k8s k8s k8s
+~~~
+
```
Comment (jalletto) [inline blog/_posts/2023-06-10-linux-text-processing-commands.md:518]:
```suggestion
<div class="notice--info">
We have a whole post on [Using sed for Find and Replace](/blog/sed-find-replace/) if you'd like to learn more.
</div>
```

File: blog/_posts/2023-06-10-linux-text-processing-commands.md [hunk 4]
```diff
@@ -0,0 +1,1019 @@
...
+~~~{ caption="Output"}
+97 markdown.md
+~~~
+
+### The `tee` Command
+
+The `tee` command allows you to redirect the output of an operation to multiple streams simultaneously, such as the `stdout` and a file. This allows you to view the output in real-time and save it for future use.
+
+A use case can be in inspecting your [container](/blog/docker-slim) image:
```
Comment (jalletto) [inline blog/_posts/2023-06-10-linux-text-processing-commands.md:734]:
```suggestion
A use case can be in inspecting your container image:
```

**Review by jalletto — CHANGES_REQUESTED** [review]

File: blog/_posts/2023-06-10-linux-text-processing-commands.md (hunk 3 above)
Comment (msali123) [reply blog/_posts/2023-06-10-linux-text-processing-commands.md:518]: image

@jalletto seems good

File: blog/_posts/2023-06-10-linux-text-processing-commands.md (hunk 2 above)
Comment (msali123) [reply blog/_posts/2023-06-10-linux-text-processing-commands.md:621]: image

Preview image

**Comment by msali123** [comment]
@jalletto [Preview Link](https://6438725ca4c56128a20d5d85--ephemeral-narwhal-3ac3b2.netlify.app/blog/linux-text-processing-commands/)

**Review by jalletto — APPROVED** [review]

**Pull request merged**